pallets/msa/src/lib.rs

```diff
@@ -117,12 +129,26 @@ pub mod pallet {
 		/// The overarching event type.
 		type RuntimeEvent: From<Event<Self>> + IsType<<Self as frame_system::Config>::RuntimeEvent>;
 
+		/// The runtime call dispatch type.
+		type Proposal: Parameter
+			+ Dispatchable<RuntimeOrigin = Self::RuntimeOrigin, PostInfo = PostDispatchInfo>
+			+ From<Call<Self>>
+			+ IsSubType<Call<Self>>
```

Comment:
I do not think you are using this trait.

Reply:
Which trait?  Do you mean the associated type Proposal (which is being used)?

Reply:
removed